src/components/TrustedTimestamping/FaqModal.vue

```diff
@@ -16,7 +16,7 @@
               img.no-border(
                 alt="Vue logo"
```

Comment:
We prolly don't need alt text here, but if we do let's align it with the end logo we use (moontography or stellar or something instead of `Vue logo`